src/main/java/seedu/mark/logic/parser/ParserUtil.java

```diff
@@ -100,6 +102,79 @@ public static Url parseUrl(String url) throws ParseException {
         return new Url(trimmedUrl);
     }
 
+    /**
+     * Parses a {@code String arg} into a {@code Tab}.
+     * Parsing will be successful only if {@code arg} is either "1", "2" or "3".
+     *
+     * @param arg The argument of a tab command
+     * @return The corresponding tab
+     * @throws ParseException if the given {@code arg} is invalid.
+     */
+    public static TabCommand.Tab parseTabIndex(String arg) throws ParseException {
```

Comment:
If `Tab` is not moved to a separate class, would it be better to use an import statement for `seedu.mark.commands.TabCommand.Tab` in this file?